lib/streamlit/server/server.py

```diff
@@ -434,6 +431,9 @@ def _create_app(self) -> tornado.web.Application:
             routes,
             cookie_secret=config.get_option("server.cookieSecret"),
             xsrf_cookies=config.get_option("server.enableXsrfProtection"),
+            # Set the websocket message size. The default value is too low.
+            websocket_max_message_size=config.get_option("server.maxMessageSize")
+            * int(1e6),
```

Comment:
This formatting looks a bit strange to me (but it seems like the auto-formatter wants it to look like this 🤷).

Maybe it'd be worth it to define `websocket_max_message_size` as a variable on the line above the `return` statement to avoid the formatting weirdness here